### DISCUSSION TITLE
Unable to set private key file: 'demo-dsc.crt' type PEM

### DISCUSSION THREAD

**Discussion by IngoWolf** [Digitaler-Impfnachweis/certification-apis#56] [Q&A]
Hi,
I'm integrating the api in our medicinal software (Vertinex Fabiola) and can't get beyond the client certificate issue. We need authentication with client-certificate (mTLS). To get the single pieces right I try to get an correct api request with curl.

This call to curl returns error 58:

curl --location --request POST 'https://api.certify.demo.ubirch.com/api/certify/v2/issue' --cert demo-dsc.crt --header 'Accept: application/cbor+base45' --header 'Content-Type: application/json' --data-raw '{"nam": {"fn": "Musterfrau","gn": "Erika"},"dob": "1979-04-14","v": [{"id": "IZ12345A","tg": "840539006","vp": "1119305005","mp": "EU/1/20/1528","ma": "ORG-100001699","dn": 1,"sd": 2,"dt": "2021-04-14"}]}'

curl: (58) unable to set private key file: 'demo-dsc.crt' type PEM

Is the given demo certificate file "demo-dsc.crt" broken? I tried many calls to openssl and can't get a private key out of it. After several hours of trying and reading discussions I now ask for help: what certificate file can I use to sucessfully connect to the api? Do I need to generate one of my own?

THX
Ingo Wolf

**Comment by thinkberg** [comment]
Please send a request to dgc-support@ubirch.com:

Subject: BMG DEMO X.509 Yourcompany (PVS)

**Reply by IngoWolf** [reply]
Many thanks. I sent the request last thursday (03.06.2021) and got no reply (today is tuesday, 08.06.2021). Our customers are eagerly waiting...
